src/components/Checkbox/Checkbox.jsx

```diff
@@ -85,6 +89,8 @@ const Checkbox = createClass({
 			...passThroughs
 		} = this.props;
 
+		const { title } = passThroughs;
```

Comment:
why not just add to the destructure statement above?

Reply:
because we still want to pass it to the input

Reply:
i think it still gets omitted by `omitProps`, since it's declared in proptypes.

Reply:
you're right, let me look into it